docs/developer/quickstart.md

```diff
@@ -46,10 +46,10 @@ This command will build the binary and output it to the `/build/.out` directory.
 To build an NGINX Kubernetes Gateway container image from source run the following make command:
 
 ```shell
-make TAG=$(whoami) container
+make container
```

Comment:
I like using the TAG here because it's clear that the image is different from edge.

Reply:
this makes sense. I brought it back